include/exiv2/types.hpp

```diff
@@ -242,7 +242,7 @@ namespace Exiv2 {
         void free();
 
         //! Reset value
-        void reset(std::pair<byte*, size_t> =std::make_pair((byte*)(0),size_t(0)));
+        void reset(std::pair<byte*, size_t> =std::make_pair((byte*)nullptr,size_t(0)));
```

Comment:
It should be possible to remove from here the cast `(byte *)` from inside the make_pair call, no ?

Reply:
Beats me. This was an automated change. I am also unfamiliar with this codebase.

Reply:
No problem, I realised about that later. Let's merge the PR as it is, since it is a useful contribution 😉